src/Moryx.TestModule/ModuleController/ModuleController.cs

```diff
@@ -35,13 +35,14 @@ public ModuleController(IOptions<ModuleConfig> options, IModuleContainerFactory
         #region State transition
 
         /// <inheritdoc />
-        protected override void OnInitialize()
+        protected override Task OnInitializeAsync()
         {
             Container.ActivateDbContexts(ContextManager);
+            return Task.CompletedTask;
         }
 
         /// <inheritdoc />
-        protected override void OnStart()
+        protected override Task OnStartAsync()
         {
             Thread.Sleep(Config.SleepTime); // Just for system testing.
```

Comment:
You could change this as you did in the `DependantTestModule`. Sorry.. hadn't you changed it once, I wouldn't even have realized probably 😅